src/azure-cli/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -149,6 +149,10 @@ def _complex_params(command_group):
         with self.argument_context('{} server vnet-rule create'.format(command_group)) as c:
             c.extra('vnet_name', options_list=['--vnet-name'], help='The virtual network name', validator=validate_subnet)
 
+        with self.argument_context('{} server vnet-rule update'.format(command_group)) as c:
+            c.extra('vnet_name', options_list=['--vnet-name'], help='The virtual network name',
```

Comment:
Is the param name --vnet-name or just --vnet ? Spec had it --vnet, so does create.